pkgs/bazel_worker/pubspec.yaml

```diff
@@ -1,16 +1,19 @@
 name: bazel_worker
-version: 1.1.3
+version: 1.1.4
 description: >-
   Protocol and utilities to implement or invoke persistent bazel workers.
 repository: https://github.com/dart-lang/tools/tree/main/pkgs/bazel_worker
 issue_tracker: https://github.com/dart-lang/tools/issues?q=is%3Aissue+is%3Aopen+label%3Apackage%3Abazel_worker
 
+workspace:
```

Comment:
Interesting - this pubspec is both a regular package pubspec as well as one that defines a workspace? I didn't know that was a thing. Perhaps add a comment above this line; `'Using a workspace so the integration tests resolve to the local version of this package.'`.